src/main/webapp/event-form.html

```diff
@@ -4,32 +4,49 @@
     <meta charset="UTF-8">
     <link rel="stylesheet" href="/css/main.css">
     <script src="/js/navigation-loader.js"></script>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="https://code.getmdl.io/1.3.0/material.indigo-pink.min.css">
+    <script defer src="https://code.getmdl.io/1.3.0/material.min.js"></script>
   </head>
   <body onload="addLoginOrLogoutLinkToNavigation();">
     <nav>
       <ul id="navigation">
         <li><a href="/">Home</a></li>
       </ul>
     </nav>
-    <h2>Create your event!</h2>
+    <h3>Create your event!</h3>
     <form action="/event-form" method="POST">
-      Name:<br>
-      <input name="eventName">
+      <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
+        <input class="mdl-textfield__input" type="text" id="eventName" name="eventName">
+        <label class="mdl-textfield__label" for="eventName">Name:</label>
+      </div>
       <br>
-      Description:<br>
-      <input name="description">
+      <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
+        <input class="mdl-textfield__input" type="text" id="description" name="description">
+        <label class="mdl-textfield__label" for="description">Description:</label>
+      </div>
       <br>
-      Organizer(s):<br>
-      <input name="organizerNames">
+      <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
+        <input class="mdl-textfield__input" type="text" id="organizerNames" name="organizerNames">
+        <label class="mdl-textfield__label" for="organizerNames">Organizer(s):</label>
+      </div>
       <br>
-      Date:<br>
-      <input name="eventDate">
+      <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
+        <input class="mdl-textfield__input" type="text" id="eventDate" pattern="[0-9/b]+" name="eventDate">
```

Comment:
You can actually get more specific too if you want.  Using `pattern="\d{1,2}/\d{1,2}/(20)?\d{2}"` require the input to be 1-2 digits, slash, 1-2 digits, slash, optional "20", and finally 2 more digits.